crates/proof-of-sql-planner/src/df_util.rs

```diff
@@ -0,0 +1,32 @@
+use arrow::datatypes::{DataType, Field, Schema};
+use datafusion::{
+    catalog::TableReference,
+    common::{Column, DFSchema},
+    logical_expr::Expr,
+};
+
+/// Create a `Expr::Column` from namespace, table, and column
+pub(crate) fn df_column(namespace: Option<&str>, table: &str, column: &str) -> Expr {
+    let full_table_name = if let Some(namespace) = namespace {
+        format!("{namespace}.{table}")
+    } else {
+        table.to_string()
+    };
+    Expr::Column(Column::new(
+        Some(TableReference::from(&full_table_name)),
+        column.to_string(),
+    ))
+}
```

Comment:
NIT:
I feel like this would be nicer to use, and it's shorter
```suggestion
pub(crate) fn df_column(table_name: &str, column: &str) -> Expr {
    Expr::Column(Column::new(
        Some(TableReference::from(table_name)),
        column.to_string(),
    ))
}
```